### PR TITLE
Clickable links in messages

### PR DESCRIPTION
**Pull request by MonsieurLanza**
This simple proposal uses ~https://github.com/alexcorvi/anchorme.js/tree/gh-pages/src~ linkifyjs to add clickable links to plain text messages. 

Could be used to make custom intern `mailto` links with some additions.

### CODE REVIEW

File: src/frontend/web_application/src/scenes/Discussion/components/MailMessage/index.jsx [hunk 1]
```diff
@@ -57,16 +58,11 @@ class MailMessage extends Component {
...
     if (!isLocked) {
-      if (!message.body_is_plain) {
-        return (
-          <TextBlock nowrap={false} className="s-mail-message__content" dangerouslySetInnerHTML={{ __html: message.body }} />
-        );
-      }
-
       return (
-        <TextBlock nowrap={false}><pre className="s-mail-message__content">{message.body}</pre></TextBlock>
+        <TextBlock nowrap={false} className="s-mail-message__content" dangerouslySetInnerHTML={{ __html: body }} />
```
Comment (iamdey) [inline src/frontend/web_application/src/scenes/Discussion/components/MailMessage/index.jsx]: actually yes, I think it's too dangerous

Comment (iamdey) [reply src/frontend/web_application/src/scenes/Discussion/components/MailMessage/index.jsx]: try to receive a plain text message w/ the following and see what happens:

```
<script>alert(1)</script>
```

Comment (MonsieurLanza) [reply src/frontend/web_application/src/scenes/Discussion/components/MailMessage/index.jsx]: Yes I understand the problem.

**Comment by MonsieurLanza** [comment]
Now uses https://soapbox.github.io/linkifyjs 

This library will escape HTML, except the links, it creates.

**Pull request merged**